src/Widgets/LogoWidget/LogoWidgetComponent.tsx

```diff
@@ -0,0 +1,19 @@
+import { Obj, provideComponent, WidgetTag } from 'scrivito'
+import { LogoWidget } from './LogoWidgetClass'
+import { isHomepage } from '../../Objs/Homepage/HomepageObjClass'
+import { Brand } from '../TopNavigationWidget/SubComponents/Brand'
+
+provideComponent(LogoWidget, ({ widget }) => {
+  const root = Obj.root()
+  if (!isHomepage(root)) return null
+
+  return (
+    <WidgetTag className="d-flex pb-4">
+      <Brand
+        root={root}
+        linkTo={widget.get('brandLink')}
+        linkClassName="navbar-brand m-auto pt-3"
+      />
+    </WidgetTag>
```

Comment:
* [ ] FYI If I look at the implementation of LogoWidget/Brand and ImageWidget it feels like ImageWidget is very close to being versatile enough to mimic the Logo behavior. We could go the general purpose route here and favor a generic ImageWidget over a specialized LogoWidget.

Reply:
Very interesting idea. I just tried it out with the existing `DataImageWidget`. Sadly `Dark logo` is not listed in the attributes.

<img width="346" alt="Screenshot 2025-03-10 at 15 47 39" src="https://github.com/user-attachments/assets/e64d5dd4-6f99-4394-a986-a54d70cfffed" />

But can be added manually by entering `siteLogoDark`:
<img width="350" alt="Screenshot 2025-03-10 at 15 49 37" src="https://github.com/user-attachments/assets/c856e98e-d77b-4d25-8075-1f681923d5bf" />